golem/task/result/resultpackage.py

```
@@ -141,6 +141,25 @@ def package_name(cls, file_path):
             return file_path
         return file_path + '.zip'
 
+    @staticmethod
+    def zip_append(ob, path, rel=""):
```

I have following comments regarding this method:
1) Parameter names are not explanatory to me. I could guess that `path` points to the file/directory we want to add to a zip archive and `ob` is probably some object representing this archive but it's not obvious. Type annotations would help too. And the role of `rel` argument is still not clear to me after reading the code.
2) Are directory paths with trailing slashes handled well? As far as I can remember `os.path.basename()` returns empty string when called on such path.
3) This method could be unit-tested by creating some temporary files/directories and mocking `ob`. Please add some unit tests. Apart from assuring correctness they would help to understand the function's desired behaviour.
4) I'm not sure if silently skipping non-existent paths is a good choice. I'd rather go with raising an exception or at least displaying a warning.

---

Thanks for comments. 

Ad 1 
As suggested I refactored names `ob` to `archive` and `rel` to `subdirectory`. Is the `rel` paramter clear now? 

Ad 2
Fixed that with `rstrip('/')`. I am not sure if I should put this inside `zip_append` function or leave it where it is now. 

Ad 3 
Added 

Ad 4 
True, I added `RuntimeException` with appropriate explanation